src/test/cpp/gtester/GeoPosition.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+#include <vector>
+#include <gtest/gtest.h>
+#include <geo/GeoPosition.h>
+
+namespace Tests {
+
+TEST(GeoPosition, SampleCalculation)
+{
+    using namespace stride::geo;
+    const GeoPosition amsterdam{52.37, 4.90};
+    const GeoPosition paris{48.86, 2.35};
+	ASSERT_NEAR(amsterdam.Distance(paris), 429.7, 1.0);
```

Comment:
The `ASSERT_NEAR`'s indentation doesn't seem quite right. Has this file been `clang-format`ted?